extension/src/extension.ts

```diff
@@ -80,6 +80,13 @@ export async function activate(context: ExtensionContext) {
         const uri = document.uri;
         testResourceManager.setDirty(uri);
         onDidChange.fire();
+        const fileName = path.basename(uri.fsPath).toLowerCase();
+        if (fileName === 'pom.xml' || fileName === 'build.gradle') {
```

Comment:
Is this reliable way to monitor the language server event? We should better off to use the language server side build_complete for this. THe server can disable auto build. I

Reply:
I think this pr should be a walkaround. In the long term, if language server doesn't export such event, we might need to open a feature request or pr to them to add this as I think it is also useful for other extensions that need such language server support.